### PR TITLE
Update DRF to 2.4.x (bug 1136676)

### PR DESCRIPTION
**Pull request by diox**
https://bugzilla.mozilla.org/show_bug.cgi?id=1136676
https://bugzilla.mozilla.org/show_bug.cgi?id=846349

### CODE REVIEW

File: mkt/feed/tests/test_serializers.py [hunk 1]
```diff
@@ -434,13 +446,15 @@ def setUp(self):
...
+        self.context = {
             'app_map': self.app_map,
             'feed_element_map': self.feed_element_map,
             'request': mkt.site.tests.req_factory_factory('')
-        }, many=True).data
+        }
+
+    def test_deserialize_many(self):
+        data = serializers.FeedItemESSerializer(
+            self.data_es, context=self.context, many=True).data
```
Comment (robhudson) [inline mkt/feed/tests/test_serializers.py]: Lots of test munging but it seems like mostly managing the context sent to the serializer.

Comment (diox) [inline mkt/feed/tests/test_serializers.py]: Yeah, some of the tests were missing a context so I rewrote them all to use the same way of passing context to spot errors more easily.

**Comment by diox** [comment]
r?

**Comment by robhudson** [comment]
Oh yeah, sorry. r+

**Comment by diox** [comment]
I'll merge this after the tag.

**Pull request merged**